crt_portal/cts_forms/models.py

```diff
@@ -127,7 +127,11 @@ def assign_section(self):
 
         if self.primary_complaint == 'voting' and 'Disability (including temporary or recovery)' not in protected_classes:
             return 'VOT'
+        elif self.primary_complaint == 'workplace' and 'Disability (including temporary or recovery)' in protected_classes:
+            return 'DRS'
         elif self.primary_complaint == 'workplace' and self.__has_immigration_protected_classes(protected_classes):
             return 'IER'
+        elif self.primary_complaint == 'workplace':
+            return 'ELS'
```

Comment:
We wouldn't want things to get too nested but in this case, I think it makes sense to skip over the conditions if we know the primary complaint isn't workplace related. I find it slightly easier to read.
```
elif self.primary_complaint == 'workplace':
    if 'Disability (including temporary or recovery)' in protected_classes:
            return 'DRS'
    elif self.__has_immigration_protected_classes(protected_classes):
            return 'IER'
    else:
            return 'ELS'
```

However, the DLS ticket doesn't look like it is close to being ready https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/72

So for now, I would write it as:
```
elif self.primary_complaint == 'workplace':
    if self.__has_immigration_protected_classes(protected_classes):
            return 'IER'
    elif 'Disability (including temporary or recovery)' not in protected_classes:
            return 'ELS'
```

Reply:
Gotcha, didnt realize there were so many conditionals there. Some of the routing logic was reflected in the ticket. I asked for guidance in the ticket, and figured I would add it since it seemed straightforward and didn't add any time to the development.  I'll go ahead and remove it!